src/lib/db/parties.ts

```diff
@@ -70,30 +70,24 @@ export async function getParty(id: string): Promise<PartyWithPersons | null> {
                                 }
                             }
                         }
-                    },
-                    where: {
-                        OR: [
-                            // Both dates are null (ongoing role)
-                            { startDate: null, endDate: null },
-                            // Only start date is set and it's in the past
-                            { startDate: { lte: new Date() }, endDate: null },
-                            // Only end date is set and it's in the future
-                            { startDate: null, endDate: { gt: new Date() } },
-                            // Both dates are set and current time is within range
-                            {
-                                startDate: { lte: new Date() },
-                                endDate: { gt: new Date() }
-                            }
-                        ]
                     }
+                    // Removed the where clause to include all roles (active and inactive)
+                    // The component will filter them appropriately
```

Comment:
Can you clarify why this is needed and what the benefit is of doing this filtering on the client side?

My concern is that inactive people aren’t really part of the Party anymore, so returning them at all feels a bit misleading. Would it make sense to model this explicitly with a flag instead?

Also, applying this change to `getParty` but not to `getPartiesForCity` could lead to inconsistent behavior and potential confusion down the line. It might be worth aligning the two APIs or documenting the difference.